scripts/etc/init.d/rcS

```diff
@@ -27,11 +27,14 @@ locales=
 system_default_locale=
 disable_predictable_nin=1
 ifname=eth0
-ip_addr=dhcp
-ip_netmask=0.0.0.0
-ip_broadcast=0.0.0.0
```

Comment:
Yes, and also it's incredibly rare for any network to have a non-default broadcast IP address.